src/utils/keymgr/functions.c

```diff
@@ -198,8 +198,8 @@ static int check_timers(const knot_kasp_key_timing_t *t)
 // modifies ctx->policy options, so don't do anything afterwards !
 int keymgr_generate_key(kdnssec_ctx_t *ctx, int argc, char *argv[])
 {
-	knot_time_t now = knot_time(), infty = 0;
-	knot_kasp_key_timing_t gen_timing = { now, infty, now, infty, now, infty, infty, infty, infty };
+	knot_time_t now = knot_time(), infinity = 0;
+	knot_kasp_key_timing_t gen_timing = { now, infinity, now, infinity, now, infinity, infinity, infinity, infinity };
```

Comment:
I'm not actually sure about this word.

Reply:
It's just an abbrev. to be shorter. I would keep the original one.